platform/darwin/src/MGLStyleValue.h

```diff
@@ -2,9 +2,27 @@
 #import <CoreGraphics/CoreGraphics.h>
 
 #import "MGLFoundation.h"
+#import "MGLTypes.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+// TODO: API docs
+typedef NSString *MGLStyleFunctionOption NS_STRING_ENUM;
+
+// TODO: API docs
+extern MGL_EXPORT const MGLStyleFunctionOption MGLStyleFunctionOptionInterpolationBase;
+
+// TODO: API docs
+extern MGL_EXPORT const MGLStyleFunctionOption MGLStyleFunctionOptionDefaultValue;
+
+// TODO: API docs
+typedef NS_ENUM(NSUInteger, MGLStyleFunctionStopType) {
```

Comment:
What term would refer to UIColor in `MGLCameraStyleValue<UIColor>`? No matter what we call it, I think developers would gravitate toward “stop type” or “type of stop” for that concept as well.

Instead of referring to exponential, interval, etc. as “stop types”, how about referring to them as “[interpolation modes](https://developer.apple.com/reference/spritekit/skinterpolationmode)”? `MGLInterpolationModeExponential` wouldn’t be _that_ much typing. 😛